app/src/main/java/com/example/moodtrackerproject/ui/WelcomeFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.moodtrackerproject.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import com.example.moodtrackerproject.R
+import com.example.moodtrackerproject.databinding.FragmentWelcomeScreenBinding
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class WelcomeFragment : Fragment() {
+
+    private var _binding: FragmentWelcomeScreenBinding? = null
+    private val mBinding get() = _binding!!
+    lateinit var navBar: BottomNavigationView
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentWelcomeScreenBinding.inflate(layoutInflater, container, false)
+
+        navBar = requireActivity()!!.findViewById(R.id.bottom_navigation)
+        navBar.isVisible = false
+
+        mBinding.welcomeButton.setOnClickListener {
+            val transaction = requireActivity().supportFragmentManager.beginTransaction()
```

Comment:
This is a "routing" logic as well, because you navigate to another route (screen)